openapi.json

```diff
@@ -836,6 +836,14 @@
                       }
                     }
                   },
+                  "branches": {
+                    "type": "array",
+                    "description": "The associated library branches.",
+                    "items": {
+                      "type": "string",
+                      "description": "The translated label of a branch."
```

Comment:
Please reconsider using the concept label here. 

As I see it label has little meaning outside of the Drupal community. As this is meant to be consumed by external parties I suggest using something like "name" or "title" instead. The description might even include an example if we want to clarify things.